aviary/mission/phase_builder_base.py

```diff
@@ -419,6 +419,17 @@ def _add_initial_guess_meta_data(cls, initial_guess: InitialGuess, desc=None):
         meta_data[name] = dict(
             apply_initial_guess=initial_guess.apply_initial_guess, desc=desc)
 
+    def _add_user_defined_constraints(self, phase, constraints):
+        # Add each constraint and its corresponding arguments to the phase
+        for constraint_name in constraints:
+            kwargs = constraints[constraint_name]
+            if kwargs['type'] == 'boundary':
```

Comment:
is constraints a dict? could this be:
`for contraint_name, kwargs in constraints.items():`

Reply:
changed!